lib/reply.js

```diff
@@ -405,9 +405,7 @@ function onSendEnd (reply, payload) {
     throw new FST_ERR_REP_INVALID_PAYLOAD_TYPE(typeof payload)
   }
 
-  if (!reply[kReplyHeaders]['content-length']) {
-    reply[kReplyHeaders]['content-length'] = '' + Buffer.byteLength(payload)
-  }
+  reply[kReplyHeaders]['content-length'] = '' + Buffer.byteLength(payload)
```

Comment:
I'm not really convinced with this change either. Essentially we are _always_ overriding the `content-length` independently of what the user wants. This seems a case that we should just document.

Reply:
@mcollina I tried another approach, what do think? For me document, it is a bit hard to explain.

Reply:
this can be improved using:
```js
`${Buffer.byteLength(buf)}`;
```

Bench: 
```
concat x 83,041,664 ops/sec ±0.51% (94 runs sampled)
template x 93,087,104 ops/sec ±0.92% (91 runs sampled)
```